src/geo/leaflet/leaflet_cartodb_layergroup.js

```diff
@@ -142,6 +142,9 @@ L.CartoDBGroupLayerBase = L.TileLayer.extend({
     });
   },
 
+  getAttribution: function() {
+    return cdb.core.sanitize.html(this.options.attribution);
```

Comment:
should we santize the attribution in this case? I think we should only do this if this was a view method